churaverse-plugins-client/src/churarenPlugin/churarenCorePlugin/churarenCorePlugin.ts

```diff
@@ -0,0 +1,121 @@
+import { CoreGamePlugin } from '@churaverse/game-plugin-client/domain/coreGamePlugin'
+import { CHURAREN_CONSTANTS } from './constants/churarenConstants'
+import { SocketController } from './controller/socketController'
+import { ChurarenDialogManager } from './ui/startWindow/churarenDialogManager'
+import { UpdateChurarenUiEvent } from './event/updateChurarenUiEvent'
+import { RegisterGameUiEvent } from '@churaverse/game-plugin-client/event/registerGameUiEvent'
+import { registerChurarenUi } from './ui/registerChurarenUi'
+import { CHURAREN_UI_KEYS } from './ui/defChurarenUi'
+import { ChurarenGameResult } from './types/uiTypes'
+import { BaseGamePlugin } from '@churaverse/game-plugin-client/domain/baseGamePlugin'
+
+export class ChurarenCorePlugin extends CoreGamePlugin {
+  public gameId = CHURAREN_CONSTANTS.GAME_ID
+  protected gameName = CHURAREN_CONSTANTS.GAME_NAME
+  private churarenDialogManager!: ChurarenDialogManager
+  private socketController!: SocketController
```

Comment:
この2つはオプショナルでも問題なく動作する気がします
```suggestion
  private churarenDialogManager?: ChurarenDialogManager
  private socketController?: SocketController
```

強制アンラップはクラッシュの原因になるので、明確なメリットが生まれて、かつ安全性が保証できる場合にのみ使用しましょう
今回の場合だと代入タイミングと利用タイミングが明確なので安全性はあるかもしれませんが、オプショナルチェーンによってほとんどコードが冗長にならずに強制アンラップを行わずに実行できるのでオプショナルにすべきかなと思います

```ts
// Before
this.churarenDialogManager.setGameAbortButtonText()

// After
this.churarenDialogManager?.setGameAbortButtonText()
```